torch/distributed/_tensor/ops/_math_ops.py

```diff
@@ -1005,20 +1006,25 @@ def layer_norm_bwd_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy
             out_placements = map_placements_after_reduction(
                 inp_placements, outer_dims, reduce_dims_map, "sum"
             )
-            output_specs_list.append(
-                DTensorSpec(
-                    mesh=mesh,
-                    placements=out_placements,
-                    tensor_meta=bias_src_spec.tensor_meta,
+
+            if output_mask[2]:
+                output_specs_list.append(
+                    DTensorSpec(
+                        mesh=mesh,
+                        placements=out_placements,
+                        tensor_meta=bias_src_spec.tensor_meta,
+                    )
                 )
-            )
+            else:
+                output_specs_list.append(None)
```

Comment:
ditto. refactor to `output_specs_list.append(... if output_mask[2] else ...)`

Reply:
Done!